### DISCUSSION TITLE
How to use Chip status ?

### DISCUSSION THREAD

**Discussion by vaynevayne** [mantinedev/mantine#3832] [Q&A] [answered]
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/124666577/226281590-e84407e0-8b87-4483-8d58-f2e5f277db4a.png">

i want use readonly chip like status, How to use it ?

**Comment by yverby** [comment, marked as answer]
You can use `disabled` prop or `checked` prop (`true` or `false`)

```jsx
<Chip checked />
<Chip disabled />
```
